src/types.ts

```diff
@@ -1236,6 +1236,13 @@ export interface LiveEventMessage {
   tags: SyncTag[]
 }
 
+export interface SanityQueries {}
```

Comment:
@sgulseth I've omitted the `extends QueryReturn` here. I don't think it's necessary to include it but I might be overlooking something.

Reply:
Hm. Might be beneficial for the typescript compiler to be sure that it's map when looking up the value, but we could optimize it later 🤔